layer-atlas/src/main/java/com/layer/ui/message/container/StandardMessageContainer.java

```
@@ -0,0 +1,70 @@
+package com.layer.ui.message.container;
+
+import android.content.Context;
+import android.graphics.drawable.GradientDrawable;
+import android.support.annotation.AttrRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.layer.ui.R;
+import com.layer.ui.databinding.UiStandardMessageContainerBinding;
+import com.layer.ui.message.model.MessageModel;
+import com.layer.ui.message.view.MessageView;
+
+public class StandardMessageContainer extends MessageContainer {
+
+    private UiStandardMessageContainerBinding mBinding;
+    private MessageView mMessageView;
+
+    public StandardMessageContainer(@NonNull Context context) {
+        this(context, null, 0);
+    }
+
+    public StandardMessageContainer(@NonNull Context context, @Nullable AttributeSet attrs) {
+        this(context, null, 0);
+    }
+
+    public StandardMessageContainer(@NonNull Context context, @Nullable AttributeSet attrs, @AttrRes int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        LayoutInflater inflater = LayoutInflater.from(getContext());
+        mBinding = UiStandardMessageContainerBinding.inflate(inflater, this, true);
+        mBinding.uiStandardMessageContainerTitle.setVisibility(GONE);
+        mBinding.uiStandardMessageContainerSubtitle.setVisibility(GONE);
+    }
+
+    public void setMessageView(MessageView view) {
+        mMessageView = view;
+        view.setLayoutParams(new ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+                ViewGroup.LayoutParams.MATCH_PARENT));
+        mBinding.uiStandardMessageContainerContentView.addView(view);
+    }
+
+    @Override
+    public <T extends MessageModel> void setMessageModel(T model) {
+        mMessageView.setMessageModel(model);
```

Is it possible at any point in time for setMessageModel() to be called before setMessageView? We may want to have a null check here

---

If someone calls `setMessageModel()` before setting a view, then it _should_ fail.  The resulting exception would warn the developer that they've done something wrong. A null check that eats the error would leave them with no indication as to what is going on.